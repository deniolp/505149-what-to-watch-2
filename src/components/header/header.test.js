import React from 'react';
import renderer from 'react-test-renderer';

import Header from './header';

describe(`Header`, () => {
  it(`renders correctly`, () => {
    window.history.pushState({}, ``, `/film/1`);
    const tree = renderer.create(
        <Header/>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
