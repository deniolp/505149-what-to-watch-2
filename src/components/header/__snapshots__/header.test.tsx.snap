// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
Array [
  <h1
    className="visually-hidden"
  >
    WTW
  </h1>,
  <header
    className="page-header movie-card__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="user-block"
    >
      <div
        className="user-block__avatar"
      >
        <a
          href="/mylist"
          onClick={[Function]}
          style={
            Object {
              "color": "#dfcf77",
              "textDecoration": "none",
            }
          }
        >
          <img
            alt="User avatar"
            height="63"
            src="https://htmlacademy-react-2.appspot.com/img/photo.jpg"
            width="63"
          />
        </a>
      </div>
    </div>
  </header>,
]
`;
